Snakefile

```diff
@@ -153,21 +155,113 @@ rule iq_tree:
         rooted="output/strains/{strain}/besttree.nwk",
         unrooted=temp("output/strains/{strain}/besttree.unrooted.treefile"),
         iqtree=temp("output/strains/{strain}/besttree.unrooted.iqtree"),
-        ckp=temp("output/strains/{strain}/besttree.unrooted.ckp.gz")
+        ckp=temp("output/strains/{strain}/besttree.unrooted.ckp.gz"),
     log:
         "output/strains/{strain}/besttree.unrooted.log"
     params:
         enabled=config['iqtree']['enabled'],
         mode=config['iqtree']['mode'],
         model=config['iqtree']['model'],
-        prefix="output/strains/{strain}/besttree.unrooted"
+        prefix="output/strains/{strain}/besttree.unrooted",
+        temp="output/strains/{strain}/temp_align_variants.aln"
```

Comment:
I don't think this is used/needed

Reply:
depends on if I remove or un-comment the other code from above

Reply:
temp is always deleted I think, so should remove